### PR TITLE
Add device-type for Aruba Mobility controller 3200

### PR DESCRIPTION
**Pull request by CedricMoreau**
Add device-type for Aruba Mobility controller 3200 (4 ports gig0/0/0 to 0/0/3 dual personnality (Base T or Base X), 1 console port, 1 power supply)

### CODE REVIEW

File: device-types/Aruba/ArubaMC-3200.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - name: (BASE-X)gigabitethernet 0/0/1
+    type: 1000base-x-sfp
+  - name: (BASE-T)gigabitethernet 0/0/2
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/2
+    type: 1000base-x-sfp
+  - name: (BASE-T)gigabitethernet 0/0/3
+    type: 1000base-t
+  - name: (BASE-X)gigabitethernet 0/0/3
+    type: 1000base-x-sfp
```
Comment (nahun) [inline device-types/Aruba/ArubaMC-3200.yaml:22]:
```suggestion
  - name: gigabitethernet 1/0
    type: 1000base-t
  - name: gigabitethernet 1/1
    type: 1000base-t
  - name: gigabitethernet 1/2
    type: 1000base-t
  - name: gigabitethernet 1/3
    type: 1000base-t
```
Only list the default personality, not both for dual personality ports. Also my 3200s have the interfaces as 1/0, 1/1, 1/2, and 1/3.

**Review by nahun — COMMENTED** [review]
Hopefully these are the revisions needed

File: device-types/Aruba/ArubaMC-3200.yaml (hunk 1 above)
Comment (alagoutte) [reply device-types/Aruba/ArubaMC-3200.yaml:22]: it is complicated about "default personality" ;-) you are sure about interface ?

Comment (nahun) [reply device-types/Aruba/ArubaMC-3200.yaml:22]: I'm just going off my 3200s for the interface names.
```
#show inventory 

Supervisor Card slot            : 0
System Serial#                  : redacted
SC      Assembly#               : 2010050C (Rev:10.00) 
SC      Serial#                 : redacted (Date:01/13/13) 
SC      Model#                  : Aruba3200-IL
<snip>
```
```
#show running-config | include 'interface gigabit' 
Building Configuration...
interface gigabitethernet 1/0
interface gigabitethernet 1/1
interface gigabitethernet 1/2
interface gigabitethernet 1/3
```

**Pull request merged**